### ISSUE TITLE
`deno compile` based `ghjk` executable

### ISSUE THREAD

**Issue by Yohe-Am** [metatypedev/ghjk#13]
This is currently blocked upstream on [#18327](https://github.com/denoland/deno/issues/18327): Deno compile doesn't currently implement dynamic imports which we make use of to load the users `ghjk.ts` and the deno worker based ports.

**Comment by zifeo** [comment]
@Yohe-Am did you give a chance to the latest comment [here](https://github.com/denoland/deno/issues/18327#issuecomment-1830255603)?

**Comment by Yohe-Am** [comment]
Yeah, looks neat. But like we discussed elsewhere, seems like the deno team is working on tackling this. What's more, that shim doesn't seem to cache the modules to `DENO_DIR` which is not ideal since we import all port impls dynamically in the core

**Comment by Yohe-Am** [comment]
This is no longer relevant since ghjk embeds deno directly.
